pkg/projectfile/projectfile.go

```diff
@@ -34,7 +34,7 @@ import (
 )
 
 var (
-	urlProjectRegexStr = fmt.Sprintf(`https:\/\/[\w\.]+\/([\w_.-]*)\/([\w_.-]*)(?:\?commitID=)*(.*)`)
+	urlProjectRegexStr = fmt.Sprintf(`https:\/\/[\w\.]+\/([\w_.-]*)\/([\w_.-]*)(?:\?commitID=)*([^&]*)(?:\&branchName=)*(.*)`)
```

Comment:
Not in the spec (it's shown as "config_name"). Since using the branch, I thought "branchName" matched "commitID" better than "branch" alone, and it namespaces a bit in case "branchID" is ever added.

I'll commit your suggestion.